libs/langchain/langchain/vectorstores/epsilla.py

```diff
@@ -0,0 +1,357 @@
+"""Wrapper around Epsilla vector database."""
+from __future__ import annotations
+
+import logging
+import uuid
+from typing import TYPE_CHECKING, Any, Iterable, List, Optional, Type
+
+from langchain.docstore.document import Document
+from langchain.embeddings.base import Embeddings
+from langchain.vectorstores.base import VectorStore
+
+if TYPE_CHECKING:
+    from pyepsilla import vectordb
+
+logger = logging.getLogger()
+
+
+class Epsilla(VectorStore):
+    """
+    Wrapper around Epsilla vector database.
+    To use you should have the ``pyepsilla`` package installed
+    and a running Epsilla vector database.
```

Comment:
and have a running Epsilla vector database (for example, through our docker image)